src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/ScheduleItem.kt

```diff
@@ -91,4 +91,6 @@ data class ScheduleItem(
   val notesForStaff: String?,
 
   val notesForPrisoners: String?,
+
+  val hmctsNumber: String?,
```

Comment:
We don't want the guest PIN here.  We only return it on the VideoLinkBooking model response e.g. for editing.

Reply:
For daily schedule maybe? Not sure whether in print view or not - maybe not, as remember Jess saying they can only view it on-screen

Reply:
Hmm OK.  Can add it needed.